app/libs/pending_comment.py

```diff
@@ -86,6 +86,21 @@ def approve(id, actor: user_id):
         action=comment_actions.approved.value,
         actor=actor
     )
+    if settings.EMAIL_NOTIFICATION:
+        commenter = memberlib.get(pending_comment['commenter_id'])
```

Comment:
@anjali-92 from here just send signal `on_comment_approved` and do all manipulations in tasks